### DISCUSSION TITLE
Homepage source

### DISCUSSION THREAD

**Discussion by SZRabinowitz** [razonyang/hugo-theme-bootstrap#1025] [General]
I am trying to find the homepage of https://hbs.razonyang.com/, but cannot find the source. Is it open source, and if so, where is it? 

When I use the exampleSite from the repository, that homepage is not included.

**Comment by razonyang** [comment]
The homepage source is located in https://github.com/razonyang/hugo-theme-bootstrap-site, it just includes the homepage, nothing more.

**Reply by SZRabinowitz** [reply]
Thanks, That's exactly what I was looking for!
